CLOUD-EDGE-CLIENT_COORDINATION_CG_Proposal

```diff
@@ -0,0 +1,11 @@
+1、Group name
+CLOUD-EDGE-CLIENT COORDINATION COMMUNITY GROUP
+
+2、Group Description
+The mission of this group is to provide mechanisms and interfaces between Central Cloud, Edge-Cloud, and Client for computing workload offloading and orchestration. 
+The typical use cases include AI acceleration, cloud gaming, streaming acceleration, etc. The goal is to design a set of new APIs and mechanisms 
```

Comment:
It's redundant to use both "including" and "etc.".  Also etc. sounds indecisive.  Suggest the following wording change.   
I have not changed this, but perhaps should also talk about what we mean by "design".   Maybe we should say "explore" or "explore and evaluate designs for" since we won't be producing specs, but (perhaps) proposals for specs, and the work will be exploratory in nature.  As a CG, we should sound as inviting as possible to accept proposals and input.

```suggestion
The typical use cases include AI acceleration, cloud gaming, and streaming acceleration. The goal is to design a set of new APIs and mechanisms 
```